<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Help</class>
 <widget class="QDialog" name="Help">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Help</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/Res/img/Help.png</normaloff>:/Res/img/Help.png</iconset>
  </property>
  <widget class="QLabel" name="lbl_BG">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>600</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(91, 14, 45);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_border">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>15</y>
     <width>770</width>
     <height>570</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 4px solid rgb(255, 167, 129);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>70</y>
     <width>311</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu Condensed</family>
     <pointsize>18</pointsize>
     <weight>9</weight>
     <italic>true</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 italic 18pt &quot;Ubuntu Condensed&quot;;
color: rgb(255, 167, 129);</string>
   </property>
   <property name="text">
    <string>Poorya Mohammadi Nasab</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>210</y>
     <width>311</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu Condensed</family>
     <pointsize>18</pointsize>
     <weight>9</weight>
     <italic>true</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 italic 18pt &quot;Ubuntu Condensed&quot;;
color: rgb(255, 167, 129);</string>
   </property>
   <property name="text">
    <string>Poorya.m.n.b@gmail.com</string>
   </property>
  </widget>
  <widget class="QLabel" name="pic_programmer">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="pic_telegram">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>360</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="pic_email2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>280</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="pic_email1">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>200</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="pic_phone">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="pic_git">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>440</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>290</y>
     <width>311</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu Condensed</family>
     <pointsize>18</pointsize>
     <weight>9</weight>
     <italic>true</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 italic 18pt &quot;Ubuntu Condensed&quot;;
color: rgb(255, 167, 129);</string>
   </property>
   <property name="text">
    <string>Poorya@gmx.com</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>370</y>
     <width>311</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu Condensed</family>
     <pointsize>18</pointsize>
     <weight>9</weight>
     <italic>true</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 italic 18pt &quot;Ubuntu Condensed&quot;;
color: rgb(255, 167, 129);</string>
   </property>
   <property name="text">
    <string>t.me/@P_M_N</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>445</y>
     <width>311</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu Condensed</family>
     <pointsize>18</pointsize>
     <weight>9</weight>
     <italic>true</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 italic 18pt &quot;Ubuntu Condensed&quot;;
color: rgb(255, 167, 129);</string>
   </property>
   <property name="text">
    <string>github.com/Pooryamn</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>130</y>
     <width>311</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu Condensed</family>
     <pointsize>18</pointsize>
     <weight>9</weight>
     <italic>true</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 italic 18pt &quot;Ubuntu Condensed&quot;;
color: rgb(255, 167, 129);</string>
   </property>
   <property name="text">
    <string>+98 913 630 7782</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
